Jenkinsfile

```diff
@@ -140,149 +128,92 @@ pipeline {
                 }
                 lock('PAYE Deployment Initiated') {
-                    colourText("info", "${env.DEPLOY_NAME}-${PAYE_TABLE}-${MODULE_NAME} deployment in progress")
+                    colourText("info", "${env.DEPLOY_TO}-${PAYE_TABLE}-${MODULE_NAME} deployment in progress")
                     deploy(PAYE_TABLE, false)
-                    colourText("success", "${env.DEPLOY_NAME}-${PAYE_TABLE}-${MODULE_NAME} Deployed.")
+                    colourText("success", "${env.DEPLOY_TO}-${PAYE_TABLE}-${MODULE_NAME} Deployed.")
                 }
-		        lock('Legal Unit Deployment Initiated') {
-                    colourText("info", "${env.DEPLOY_NAME}-${LEU_TABLE}-${MODULE_NAME} deployment in progress")
+                lock('Legal Unit Deployment Initiated') {
+                    colourText("info", "${env.DEPLOY_TO}-${LEU_TABLE}-${MODULE_NAME} deployment in progress")
                     deploy(LEU_TABLE , true)
-                    colourText("success", "${env.DEPLOY_NAME}-${LEU_TABLE}-${MODULE_NAME} Deployed.")
+                    colourText("success", "${env.DEPLOY_TO}-${LEU_TABLE}-${MODULE_NAME} Deployed.")
                 }
             }
-        }
-
-        stage ('Package and Push Artifact') {
-            agent any
-            when {
-                expression {
-                    isBranch(BRANCH_DEV) || isBranch(BRANCH_TEST)
+            post {
+                success {
+                    colourText("info","Stage: ${env.STAGE} successful!")
                 }
-            }
-            steps {
-                script {
-                    env.NODE_STAGE = "Package and Push Artifact"
+                failure {
+                    colourText("warn","Stage: ${env.STAGE} failed!")
                 }
-                sh """
-                    $SBT 'set test in assembly := {}' clean compile assembly
-                """
-                copyToHBaseNode()
-                colourText("success", 'Package.')
             }
         }
 
-        stage("Releases"){
+        stage ('Deploy HBase') {
             agent any
-            when {
-                expression {
-                    isBranch(BRANCH_DEV) || isBranch(BRANCH_TEST) || isBranch(BRANCH_PROD)
-                }
+            when{ expression{ isBranch("master") }}
+            environment {
+                STAGE = "Deploy HBase"
+                DEPLOY_TO = "dev"
             }
             steps {
-                script {
-                    STAGE = "Releases"
-                    currentTag = getLatestGitTag()
-                    colourText("info", "Found latest tag: ${currentTag}")
-                    newTag =  IncrementTag( currentTag, RELEASE_TYPE )
-                    colourText("info", "Generated new tag: ${newTag}")
-                    //push(newTag, currentTag)
-                }
+                sh "sbt 'set test in assembly := {}' assembly"
```

Comment:
Ranjit needs to fix this. This is omitting the tests when assembling the JAR. The cause I believe was that it needed to execute something in the 'tmp' folder - sounds familiar I'm sure.

Reply:
This is a pretty good optimization; we run `test` in the `Test` stage, we dont need to run it again here 👍

Reply:
Good point